tests/steps/deploy/cloudfront/test_cloudfront.py

```diff
@@ -0,0 +1,12 @@
+from mpyl.steps.deploy.cloudfront_kubernetes_deploy import CloudFrontKubernetesDeploy
+from src.mpyl.steps.models import Input
+from tests.test_resources.test_data import get_cloudfront_project, RUN_PROPERTIES
+
+
+class TestCloudFront:
```

Comment:
it's tough to unit test the execute method here - it creates directories, copies files from docker containers, and uploads data to S3. I'll spend a little more time looking at this, but any recommendations?

Reply:
I guess you can try mocking the calls and test the parameters that are passed around? Not sure if that's really feasible in python though. You can try moving as much as possible to utility functions and testing those?

Reply:
And you can check as example the other deploy step tests

Reply:
Ah I see that you mock the calls and moved everything to util methods, so maybe disregard my earlier comment :D I think it's fine like this